ci-operator/step-registry/cucushift/pre/cucushift-pre-commands.sh

```diff
@@ -65,14 +65,14 @@ runtime_env=${SHARED_DIR}/runtime_env
 
 cat <<EOF >>"${runtime_env}"
 # cucumber setting
+export USERS=${users}
+export TAG_VERSION="@${ver_cli}"
+
 export CUCUMBER_PUBLISH_QUIET=true
 export DISABLE_WAIT_PRINT=true
 
-# Export those parameters before running
-export BUSHSLICER_DEFAULT_ENVIRONMENT=ocp4
 export OPENSHIFT_ENV_OCP4_HOSTS="${hosts}:lb"
 export OPENSHIFT_ENV_OCP4_ADMIN_CREDS_SPEC=file:///tmp/kubeconfig
+export BUSHSLICER_DEFAULT_ENVIRONMENT=ocp4
```

Comment:
We need to set `BUSHSLICER_DEFAULT_ENVIRONMENT` first, the `OCP4` in `OPENSHIFT_ENV_OCP4_HOSTS` and `OPENSHIFT_ENV_OCP4_ADMIN_CREDS_SPEC` are coming from the value of this var.

Reply:
Thank you! fixed now.